torch/distributed/_sharded_tensor/api.py

```diff
@@ -550,9 +559,46 @@ def sharding_spec(self) -> ShardingSpec:
         """
         return self._sharding_spec
 
+    def reshard(self, resharding_spec: ReshardingSpec) -> Union[ShardedTensor, torch.Tensor]:
+        """
+        Reshards a tensor given the resharding_spec, if output_local_tensors is
```

Comment:
nit: Use double ticks `` to highlight `output_local_tensors`

Reply:
Sure.